go/runtime/bundle/discovery_test.go

```diff
@@ -67,13 +69,18 @@ func TestBundleDiscovery(t *testing.T) {
 	// Prepare a temporary directory for storing bundles.
 	dataDir := t.TempDir()
 
+	// Create dummy runtimeID1
+	var runtimeID1 common.Namespace
```

Comment:
```suggestion
	var runtimeID common.Namespace
```
This is actually not needed so you can remove it and just pass `nil` to the discovery.

Reply:
Thanks! 5a1b1a1801970fdfb974c387aa653f391b165f66 -> much nicer indeed